05-html-css/css-framework.md

```diff
@@ -18,185 +16,144 @@ The usual components are:
 
 ![grid](imgs/7_Grid1-530x265.jpg)
 
-Grids have long been used in design to provide structure and balance for content in a layout. Interested in learning more about grid theory? [Read this](https://designschool.canva.com/blog/grid-design/)
-
+### Bootstrap's 12-column Flexbox-based grid system
+![Bootstrap Grid System](imgs/bootstrap-grid.jpg)
 
-### Foundation uses a 12 column grid.
-![Foundation Grid](http://foundation.zurb.com/assets/img/seo/feature-grid-1.png)
+At Ada we will continue to use CSS Grid & Flexbox for layout, but it's important to know that many sites use CSS framework layouts like the 12 column grid layout.  If you want to learn more about Bootstrap's Grid system you can check [this out](http://getbootstrap.com/docs/4.1/layout/grid/).
 
-At Ada we will continue to use CSS Grid & Flexbox for layout, but it's important to know that many sites use CSS framework layouts like the 12 column grid foundation layout.  If you want to learn more about Foundation's XY-Grid you can check [this out](https://foundation.zurb.com/sites/docs/xy-grid.html).  One nice advantage to Foundation's grid system is that it can be easier to make responsive to varying screen sizes.
+## Add Bootstrap to your project
 
-## Add Foundation to your project
-
- When choosing to use Foundation in a Rails project you can select to either:
+ When choosing to use Bootstrap in a Rails project you can select to either:
  1.  Manually copy and include all the CSS, JavaScript and SCSS files into your project.
- 2. Use a provided [foundation-rails](https://github.com/zurb/foundation-rails) gem.
+ 2. Use a provided [Bootstrap](https://github.com/twbs/bootstrap-rubygem) gem.
 
-We will take the expedient of using the Gem, and placing it into our [`.rails-template.rb`](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/09-intermediate-rails/reference/.rails-template.rb) file.  
+We will take the expedient of using the Gem, and placing it into our [`.rails-template.rb`](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/09-intermediate-rails/reference/.rails-template.rb) file.
 
-In our [`rails-template.rb`](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/09-intermediate-rails/reference/.rails-template.rb) file we added code to automatically install the Foundation 6 framework.
+In our [`rails-template.rb`](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/09-intermediate-rails/reference/.rails-template.rb) file we added code to automatically install the Bootstrap 4 framework.
```

Comment:
They should already have this from the work we did on Monday morning, right? If so, probably makes sense to reduce this section to explain that this has already happened, and maybe point out how the gem has been added to the `Gemfile`, and the javascript bit as well. I don't think we need the template code copy-pasted here.